### PR TITLE
fix: do not try to process JSON files

### LINKED ISSUE #300: Bug: preview-generator tries to handle JSON files and fails
## Description and expectations

Handing a file which has the mime type 'application/json' to preview-generator results in preview-generator trying to generate a preview using wand. This, however, fails and an exception of type `wand.exceptions.MissingDelegateError` is thrown. I would expect that JSON files are not being processed in any way and that an exception of type `preview_generator.exception.UnsupportedMimeType` is thrown.

### How to reproduce

    import json
    from tempfile import NamedTemporaryFile
    from preview_generator.manager import PreviewManager

    cache_path = '/tmp/preview_cache'

    with NamedTemporaryFile(mode='w+', suffix='.json') as fp:
        json.dump({'foo': 42, 'bar': 99}, fp)

        manager = PreviewManager(cache_path, create_folder= True)
        path_to_preview_image = manager.get_jpeg_preview(fp.name)
        print(path_to_preview_image)

### Version information

- Version: 0.29
- Device / System: Debian Bullseye

### PR DESCRIPTION
**Pull request by hrittich**
Fixes #300.

### CODE REVIEW

File: preview_generator/preview/builder/image__wand.py [hunk 1]
```diff
@@ -80,6 +80,12 @@ def __load_mimetypes(cls) -> typing.List[str]:
         """
 
         mimes = imagemagick_supported_mimes()  # type: typing.List[str]
+
+        # Image magick claims to support files of type 'application/json', however we cannot
+        # generate previews for JSON files.
+        mimes.remove('application/json')
+        print(mimes)
```
Comment (raphj) [inline preview_generator/preview/builder/image__wand.py]: leftover?

Comment (hrittich) [reply preview_generator/preview/builder/image__wand.py]: Indeed...

Comment (hrittich) [reply preview_generator/preview/builder/image__wand.py]: Removed the `print` statement.

Comment (raphj) [reply preview_generator/preview/builder/image__wand.py]: I'll let Algoo handle the rest :-)

**Comment by TheWathis** [comment]
Thanks for the contribution @hrittich and thanks for the review @raphj 

Since this PR removes the support to display `JSON` (because ImageMagick doesn't support `JSON` as an input, therefore nor is Wand), if we merge this PR we also have to modify the [documentation](https://github.com/algoo/preview-generator/blob/develop/doc/supported_mimetypes.rst). If you can do that, we are ready to go!

**Review by TheWathis — CHANGES_REQUESTED** [review]
As said before, this fix need to also update the [documentation](https://github.com/algoo/preview-generator/blob/develop/doc/supported_mimetypes.rst).

**Comment by hrittich** [comment]
@Haskaris I have updated the PR to include the update for the documentation.

**Review by TheWathis — APPROVED** [review]

**Comment by TheWathis** [comment]
Working as a charm! Thank you for the contribution!

**Pull request merged**